### PR TITLE
added Page writer + test

### PR DESCRIPTION
**Pull request by dvirarad**

### CODE REVIEW

File: src/main/scala/com/pipl/dap/parser/DapParser.scala [hunk 1]
```diff
@@ -50,6 +52,15 @@ class DapParser(xml: String) {
     recordStr
   }
 
+  def toXml(page: Page): String = {
```
Comment (PiplYaki) [inline src/main/scala/com/pipl/dap/parser/DapParser.scala]: If we already transform xml to record in order to write it as page. There is no limitation to provide statistics. So lets keep the statistics

File: src/test/scala/com/pipl/dap/writer/DapWriterTest.scala [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    val url = getClass.getResource("/settings.properties")
+    System.setProperty("settings", url.getPath)
+    new ServiceManager().init(true,true,false,false,false,false,false,false)
+
+  }
+
+
+  "page writer with empty page" must "be equal on serialisation and deserialisation " in {
+    Given("empty page")
+    val xml = """<Pg V="v3t"></Pg>"""
```
Comment (PiplYaki) [inline src/test/scala/com/pipl/dap/writer/DapWriterTest.scala]: is this legit status? are we allowing empty page?

Comment (dvirarad) [reply src/test/scala/com/pipl/dap/writer/DapWriterTest.scala]: this is only separate the tests

File: src/main/scala/com/pipl/dap/parser/DapParser.scala (hunk 1 above)
Comment (dvirarad) [reply src/main/scala/com/pipl/dap/parser/DapParser.scala]: This is only for current status  that: 
1. extraction give us files with record xml.
2. Insertion FW  getting the record
3. read it as Class
4. create Page class
5. insert the record to the page
6. write the page as tiny XML.
 In the future the FW will get from extraction files with Page XML, so he can skip tasks 3-6

**Pull request merged**